arch/risc-v/src/common/riscv_assert.c

```diff
@@ -172,44 +175,65 @@ static void riscv_dump_task(struct tcb_s *tcb, void *arg)
     }
 #endif
 
+#ifndef CONFIG_DISABLE_PTHREAD
+  if ((tcb->flags & TCB_FLAG_TTYPE_MASK) == TCB_FLAG_TTYPE_PTHREAD)
+    {
+      FAR struct pthread_tcb_s *ptcb = (FAR struct pthread_tcb_s *)tcb;
+
+      snprintf(args, sizeof(args), "%p ", ptcb->arg);
+    }
+  else
+#endif
+    {
+      FAR char **argv = tcb->group->tg_info->argv + 1;
+      size_t npos = 0;
+
+      while (*argv != NULL && npos < sizeof(args))
+        {
+          npos += snprintf(args + npos, sizeof(args) - npos, "%s ", *argv++);
+        }
+    }
+
   /* Dump interesting properties of this task */
 
   _alert("  %4d   %4d"
 #ifdef CONFIG_SMP
          "  %4d"
 #endif
+         "   %7lu"
 #ifdef CONFIG_STACK_COLORATION
          "   %7lu"
 #endif
-         "   %7lu"
 #ifdef CONFIG_STACK_COLORATION
          "   %3" PRId32 ".%1" PRId32 "%%%c"
 #endif
 #ifdef CONFIG_SCHED_CPULOAD
          "   %3" PRId32 ".%01" PRId32 "%%"
 #endif
 #if CONFIG_TASK_NAME_SIZE > 0
-         "   %s"
+         "   %s %s\n"
```

Comment:
I just looked into procfs that does similar and there we have
```
#if CONFIG_TASK_NAME_SIZE > 0
  name       = tcb->name;
#else
  name       = "<noname>";
#endif
```
maybe we can use similar approach here. What do you think?
Then we can using format string to `"%s%s\n"` and print args with `" %p"`/`" %s"` instead of `"%p "`/`"%s "`